.changeset/pretty-eagles-matter.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-bazaar-backend': patch
+---
+
+Column `title` has replaced column `name` for `BazaarProject` in database
```

Comment:
I removed name altogether instead. First we wanted a BazaarProject to match the format of an entity, but since name fills no real purpose I think title is doing a good job replacing it :) Also, id already works as an identifier for a project